common/source/docs/common-radio-control-calibration.rst

```diff
@@ -1,28 +1,85 @@
+   -  A Red or flashing LED may indicate that your RC transmitter/receiver need be bound.  See the manual that came with your RC equipment for instructions
+
+.. figure:: ../../../images/mp_radio_calibration.png
+   :target: ../_images/mp_radio_calibration.png
+
+- Check the channel mapping in the transmitter (ie. check which inputs channels are controlled by the transmitter's sticks, switches and knobs) by moving the sticks, knobs and switches and observing which (if any) green bars move.  If this is the first time the transmitter is being used with ArduPilot it is likely that the transmitter's channel mapping will need to be changed and normally this is done on the transmitter itself using its built-in configuration menu
+
+  - Determine if your transmitter is Mode1 or Mode2 (see below)
+  - Roll stick should control channel 1
+  - Pitch stick should control channel 2
+  - Throttle stick should control channel 3
+  - Yaw stick should control channel 4
+  - A 3 position switch (to control the flight mode) should be setup to control Channel 5 (if using Copter) or Channel 8 (if using Rover or Plane).  This channel can be moved by setting the :ref:`MODE_CH <MODE_CH>` parameter
+  - On Copter, a tuning knob should control Channel 6
+  - On Copter and Rover, any remaining two or three position switches can be setup to control auxiliary functions by mapping them to channels 7 to 12
+- Move the transmitter's roll, pitch, throttle and yaw sticks and ensure the green bars move in the correct direction:
+
+  - for roll, throttle and yaw channels, the green bars should move in the same direction as the transmitter's physical sticks
+  - for pitch, the green bar should move in the opposite direction to the transmitter's physical stick
+  - if one of the green bars moves in the incorrect direction reverse the channel in the transmitter itself.  If it is not possible to reverse the channel in the transmitter you may reverse the channel in ArduPilot by checking the "Reversed" checkbox (Plane and Rover only).  If the checkbox is not visible it is possible to reverse the channel by directly changing the RCx_REVERSED parameter (where "x" is the input channel from 1 to 4).
+
+.. figure:: ../../../images/mp_radio_calibration.png
+   :target: ../_images/mp_radio_calibration.png
+
+Calibration
+-----------
+
+- Open Mission Planner's **INITIAL SETUP \| Mandatory Hardware \| Radio Calibration** screen
+- Click on the green "Calibrate Radio" button on the bottom right
+- Press "OK" when prompted to check the radio control equipment is on, battery is not connected, and propellers are not attached
+
+   .. figure:: ../../../images/mp_calibrate_radio.jpg
+      :target: ../_images/mp_calibrate_radio.jpg
+
+- Move the transmitter's control sticks, knobs and switches to their limits.  Red lines will appear across the calibration bars to show minimum and maximum values seen so far
+
```

Comment:
Should we discuss that fact that you should make sure that the THROTTLE FAILSAFE throttle minimum should NOT be included in this range calibration? most either use throttle low trim or a switch to force throttle below low stick value to implement this...if its included in the low throttle range (RC_MIN) of this cal it causes all sorts of issues...